atlas/atlas.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly ATLAS_VERSION='1.1.0'
```

Comment:
I have tried with 1.2.0 and it worked. Are there any reasons to not use the 1.2.0 version?